web/package.json

```diff
@@ -1,38 +1,40 @@
 {
-  "name": "web",
-  "private": true,
+  "name": "ucl-arc-tre-portal",
   "version": "0.0.0",
-  "type": "module",
+  "private": true,
   "scripts": {
-    "dev": "vite",
-    "build": "tsc -b && vite build",
+    "dev": "next dev --turbo",
+    "lint:all": "npm run lint && npm run lint:cypress",
+    "build": "next build",
+    "start": "next start",
     "lint": "eslint .",
-    "preview": "vite preview",
+    "lint:cypress": "eslint --config cypress/eslint.config.mjs cypress/",
```

Comment:
Theres a few different `lint` commands here and it is perhaps a bit convoluted. The lint command is actually already built in with `npm run lint`.

In addition, the current `"lint": "eslint ."` command will try to run the linter across all files, including the built static files that are made during `npm run build`. I don't think this is the behaviour we want.

Ideally, the linter would run across all of our own code, and ignore anything else.

Reply:
You may also not need the `lint: cypress` command if our configs are setup correctly. But I'm less sure about that.

Reply:
I followed this [article](https://blog.linotte.dev/eslint-9-next-js-935c2b6d0371) to get ESLint to work with next14 because they're not directly compatible - next15 is the one that officially works with eslint9 and eslint8 with next14 but eslint8 is no longer supported. We bypass the built in linting because it throws errors

Reply:
we have a specific command for linting cypress, which we could get rid of if we don't want to lint the contents of the cypress folder (and can remove the config for it)
the plain lint command should be used in the release build because there's no cypress reference and so shouldn't complain that cypress can't be found

Reply:
At the moment, if you run a build locally with `npm run build` it won't pass the linter check because it is checking the built folders (try it locally). You may need to add them to the config to ignore them.